concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -0,0 +1,228 @@
+from base64 import b64encode
+from datetime import datetime
+
+import mock
+from django.conf import settings
+from django.test import override_settings
+from django.urls import reverse
+from freezegun import freeze_time
+
+from golem_messages import message, load
+
+from core.message_handlers import store_or_update_subtask
+from core.models import Subtask
+from core.tests.utils import ConcentIntegrationTestCase
+from core.transfer_operations import create_file_transfer_token
+from utils.helpers import decode_key
+from utils.testing_helpers import generate_ecc_key_pair
+
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY) = generate_ecc_key_pair()
+
+
+@override_settings(
+    CONCENT_PRIVATE_KEY  = CONCENT_PRIVATE_KEY,
+    CONCENT_PUBLIC_KEY   = CONCENT_PUBLIC_KEY,
+)
+class SubtaskResultsVerifyIntegrationTest(ConcentIntegrationTestCase):
+    def setUp(self):
+        super().setUp()
+        self.task_id = "task_1"
+        self.subtask_id = "subtask_1"
+        self.report_computed_task = self._create_report_computed_task()
+
+    def test_that_concent_responds_with_service_refused_when_verification_for_this_subtask_done_before(self):
+        # given
```

Comment:
We have a non-written convention to describe message exchange in test comment. In several cases it gives very quick view of what the test actually does, please add them, check in other test_integration_*.

Reply:
done